tests/test_request_attrs.py

```diff
@@ -32,6 +31,13 @@ def _make_etag(value, is_weak=False):
     return etag
 
 
+# NOTE(vytas): create_req is very heavily used in this module in unittest-style
+#   classes, so we simply recreate the function here.
+def create_req(asgi, options=None, **environ_or_scope_kwargs):
+    create_method = testing.create_asgi_req if asgi else testing.create_req
+    return create_method(options=options, **environ_or_scope_kwargs)
+
+
```

Comment:
the `@pytest.mark.parametrize('asgi', [True, False])` applied to the class below could also be removed

Reply:
Yeah, I had it removed first but then had issues trying to port too much other stuff, and then forgot about it.
Removed now.